### ISSUE TITLE
error when navigating directly to login page

### ISSUE THREAD

**Issue by macgyver** [clay/amphora-auth#39]
not sure exactly where the bug is but it seems like if you navigate directly to `{site-domain}/_auth/login` to log in as a clay user, you get an error like this: 
![image](https://github.com/user-attachments/assets/83d69e21-96ca-40df-8613-0b4b83fdd0fb)

getting there by tacking on `?edit=true` or typing C-L-A-Y doesn't have the same issue..

can someone at nymag see if this is reproducible?

**Comment by salomoneb** [comment]
Are you still getting this @macgyver ? I'm unable to reproduce.

**Comment by macgyver** [comment]
Yes, I still am - start by navigating to, e.g. https://slate.com/_auth/login - then login with your google account. After the redirect, you will see the error message in the screenshot (just retested in Firefox and Chrome)

**Comment by salomoneb** [comment]
Ah yes, now I see.
